src/main/java/mods/railcraft/common/plugins/forge/PlayerPlugin.java

```diff
@@ -70,7 +100,7 @@ public static String getUsername(World world, UUID playerId) {
             if (player != null)
                 return player.getDisplayName();
         }
-        return "[Unknown]";
+        return "[Railcraft]";
```

Comment:
This probably breaks stuff...I remember some "[Unknown]".equals(string) somewhere.